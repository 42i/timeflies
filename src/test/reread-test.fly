wp a
day 2012-07-17 8:30 17:30
sick 3:45
day 2012-07-18 7:45 17:30, off 0:45
- a 2:15; two and a quarter
day 2012-07-19 7:45 17:30, off 0:45
leave 1:30
day 2012-07-20, phol; Goons' Day
leave-days 2012-07-21 2012-07-25; Munro bagging
sick-days 2012-07-26 2012-07-30; Sore toe
