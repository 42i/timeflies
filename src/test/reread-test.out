reread-test.fly:2: ERROR : re-defining day (this file has already been read before)
reread-test.fly:3: ERROR : re-defining sick (this file has already been read before)
reread-test.fly:4: ERROR : re-defining day (this file has already been read before)
reread-test.fly:4: ERROR : re-defining off (this file has already been read before)
reread-test.fly:6: ERROR : re-defining day (this file has already been read before)
reread-test.fly:6: ERROR : re-defining off (this file has already been read before)
reread-test.fly:7: ERROR : re-defining leave (this file has already been read before)
Time at work overview (all):
     when        worked   leave    sick balance
2012-07-17 Tue:    9.00 ----.--    3.75    4.75
2012-07-18 Wed:    9.00 ----.-- ----.--    1.00
2012-07-19 Thu:    9.00    1.50 ----.--    2.50
2012-07-20 Fri: ----.-- ----.-- ----.-- ----.-- Goons' Day
  week 2012-29:   27.00    1.50    3.75    8.25
2012-07-23 Mon: ----.--    8.00 ----.-- ----.-- Munro bagging
2012-07-24 Tue: ----.--    8.00 ----.-- ----.-- Munro bagging
2012-07-25 Wed: ----.--    8.00 ----.-- ----.-- Munro bagging
2012-07-26 Thu: ----.-- ----.--    8.00 ----.-- Sore toe
2012-07-27 Fri: ----.-- ----.--    8.00 ----.-- Sore toe
  week 2012-30: ----.--   24.00   16.00 ----.--
2012-07-30 Mon: ----.-- ----.--    8.00 ----.-- Sore toe
  week 2012-31: ----.-- ----.--    8.00 ----.--
 month 2012-07:   27.00   25.50   27.75    8.25
         total:   27.00   25.50   27.75    8.25
     when        worked   leave    sick balance
